docs/user-guide/configuring-at-tls-for-zowe-server.md

```diff
@@ -46,11 +46,24 @@ The Discovery Service endpoints are not reachable by standard API Gateway routin
 Zowe v3 includes a new component named ZAAS (Zowe Authentication and Authorization Service). In AT-TLS-aware mode, calls to this service are all internal between API ML components. These must include the X.509 Client Certificate.
 :::
 
-### Limitations
+### Limitations when using AT-TLS with ICSF Hardware keyring
 
-If using AT-TLS with a z/OS Keyring backed by an ICSF hardware module, the only supported configuration is Zowe with z/OSMF authentication provider in JWT mode.
-A LTPA token and SAF provider cannot be used in this configuration because API ML cannot access the hardware key to sign its own tokens.
-Personal Access Tokens (PAT) are not supported in this configuration because API ML cannot access the hardware key to sign the tokens.
+API Mediation Layer cannot currently read private keys if they are backed in a hardware module. When using AT-TLS with a z/OS Keyring backed by an ICSF hardware module, use one of the following options:
+
+#### 1. Prevent API Mediation Layer from reading the private key
+
+Set `environments.APIML_ATTLS_LOAD_KEYRING: true` in zowe.yaml to prevent API Mediation Layer from loading the keyring.
+The only supported configuration is Zowe with z/OSMF authentication provider in JWT mode.
+This mode requires both server and client AT-TLS enabled in zowe.yaml with full coverage of Inbound and Outbound rules.
+
+**Note:** z/OSMF LTPA token, SAF native authentication provider and Personal Access Tokens (PAT) cannot be used in this configuration due to lack of a private key.
+
+#### 2. Use an alternative non-hardware keyring
+
+Since handshakes are handled by AT-TLS, API Mediation Layer only requires a private key access to sign its own tokens when the configuration requires it:
+- Personal Access Tokens
```

Comment:
I'm a bit confused as to what this bulleted list represents? Can you please add a sentence before this list to explain what these three bulleted items represent?

Reply:
I added a statement. Please review for accuracy.